### ISSUE TITLE
Malformed imports for array overrides

### ISSUE THREAD

**Issue by aitva** [kyleconroy/sqlc#2316]
### Version

1.18.0

### What happened?

I tried to upgrade to sqlc v1.18.0 and the Go compiler reports `malformed import path` in the generated sources. I am using column overrides for array types and the generated code keeps the `[]` the import URL.

Here is the code that was working in v1.17.2: https://play.sqlc.dev/p/17e05c48de88123b99ffff7b585291a45f7db6099cbba3627888c1f238f6c6fe

Here is the same code in v1.18.0: https://play.sqlc.dev/p/5c3833f312ae1596952586f263eecc4ce56b2736aa45e90a1441e04b6a99b8db

If you look at the generated `model.go`, you will notice the `"database/[]sql"` import that produces the error.

### Playground URL

https://play.sqlc.dev/p/5c3833f312ae1596952586f263eecc4ce56b2736aa45e90a1441e04b6a99b8db

### What operating system are you using?

Linux

### What database engines are you using?

PostgreSQL

### What type of code are you generating?

Go

**Comment by andrewmbenton** [comment]
The fact that an override configuration like `"go_type": "database/[]sql.NullInt64"` ever worked is a little surprising. We just merged a change I made to add a `"slice"` boolean property to the object definition format for the override `"go_type"` property. I also added some documentation to the bottom of the Type Overriding section here: https://docs.sqlc.dev/en/latest/reference/config.html#type-overriding

Basically you just need to do something like the following in your overrides list:
```json
{
  "column": "authors.tags",
  "go_type": {
    "import": "database/sql",
    "type": "NullInt64",
    "slice": true
}
```

This should work with a build off of the main branch, and it will be in the next release.
